project/AkkaBuild.scala

```diff
@@ -129,8 +129,6 @@ object AkkaBuild {
   lazy val defaultSettings: Seq[Setting[_]] = Def.settings(
     resolverSettings,
     TestExtras.Filter.settings,
-    Protobuf.settings,
```

Comment:
Woops, will that also solve the multiple assembly'd jars on the classpath issue?

Reply:
Yes, as it was added twice on the projects that actually needed it

What I don't understand is why this causes the classes from the unmanaged jar end up in actor-actor, but not in the actual modules that apply these settings.